ts/main.ts

```diff
@@ -1 +1,41 @@
-/* global data */
+// /* global data */
+
+const $photoUrlElement = document.querySelector('#photo-url-field');
+if (!$photoUrlElement) throw new Error('the photo-url-field query failed');
+
+function getPhoto(event: any): void {
+  const eventTarget = event.target;
+  const photoUrl = eventTarget.value;
+  const $image = document.querySelector('img');
```

Comment:
`querySelector`s should usually be done globally, because we only need to do the search once. By putting it inside this function, now anytime a user ever `input`s a key, you are going to force the browser to traverse the DOM